salt/modules/nilrt_ip.py

```diff
@@ -9,30 +9,42 @@
 import logging
 import time
 import os
+import re
 
 # Import salt libs
 import salt.exceptions
 import salt.utils.files
 import salt.utils.validate.net
 
 # Import 3rd-party libs
+# pylint: disable=import-error,redefined-builtin,no-name-in-module
+from salt.ext.six.moves import map, range
 from salt.ext import six
-from salt.ext.six.moves import configparser
+# pylint: enable=import-error,redefined-builtin,no-name-in-module
 try:
     import pyconnman
-    HAS_PYCONNMAN = True
 except ImportError:
-    HAS_PYCONNMAN = False
+    pyconnman = None
+
 try:
     import dbus
-    HAS_DBUS = True
 except ImportError:
-    HAS_DBUS = False
+    dbus = None
+
 try:
     import pyiface
-    HAS_PYIFACE = True
 except ImportError:
-    HAS_PYIFACE = False
+    pyiface = None
+
+try:
+    import configparser
+except ImportError:
+    configparser = None
```

Comment:
This still needs to be imported from six.

`from salt.ext.six.moves import configparser`

Because the name changed between py2 and py3